src/main/java/com/sourceauditor/spdx_to_osv/Main.java

```diff
@@ -294,7 +297,15 @@ public static void spdxToOsv(IModelStore fromStore, String documentUri, Writer w
                 if (downloadLocation.isPresent()) {
                     Optional<OsvVulnerabilityRequest> pnv = new DownloadLocationParser(downloadLocation.get()).getOsvVulnerabilityRequest();
                     if (pnv.isPresent()) {
-                        pvSet.add(pnv.get());
+                        OsvVulnerabilityRequest req = pnv.get();
+                        if (req.getVersion() == null && req.getCommit() == null) {
+                            if (version.isPresent()) {
+                                req.setVersion(version.get());
+                            } else {
+                                continue;
+                            }
```

Comment:
@lumjjb - curious why the continue statement above - I believe it would essentially skip any requests that do not have a version.  Let me know if this is the intended behavior and why.

Reply:
Yea - so the issue i ran into here was that a request without either `commit` or `version` is considered a bad request - and will throw an exception. Another behavior we can adopt here is to default to setting it to `v0` which OSV will be able to fuzz query (although this isn't exactly accurate)

Another thing to do would be to log warnings to the user that this entry could not be queried due to lack of version/commit.

Reply:
I added a warning output in stderr.